### PR TITLE
Create FileExecutionWithOneCharacterInTheName.yaml

### PR DESCRIPTION
**Pull request by vpaschalidis**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by thmcelro** [comment]
Hi @vpaschalidis ! Thanks for your query, lets begin with the below suggested changes

- Use a double space in front of query instead of 4 spaces or tabs
- Correctly indent line 17
- Provide entity mappings for Computer, User, File and IP (where possible we provide these in hunting queries now)

File: Hunting Queries/FileExecutionWithOneCharacterInTheName.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  | parse EventData with * 'User">' User "<" *
+  | parse EventData with * 'LogonGuid">' LogonGuid "<" *
+  | parse EventData with * 'IntegrityLevel">' IntegrityLevel "<" *
+  | parse EventData with * 'Hashes">' Hashes "<" *
+  | parse EventData with * 'ParentProcessGuid">' ParentProcessGuid "<" *
+  | parse EventData with * 'ParentImage">' ParentImage "<" *
+  | parse EventData with * 'ParentCommandLine">' ParentCommandLine "<" *
+  | parse EventData with * 'ParentUser">' ParentUser "<" *
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated) by EventID, Computer, User, ParentImage, ParentProcessGuid, ParentCommandLine, ParentUser, Image, ProcessGuid, CommandLine, Description, OriginalFileName, CurrentDirectory, Hashes
+  entityMappings:
```
Comment (thmcelro) [inline Hunting Queries/FileExecutionWithOneCharacterInTheName.yaml:36]: Indentation is broken here, remove the double space from here down to line 48

Comment (vpaschalidis) [reply Hunting Queries/FileExecutionWithOneCharacterInTheName.yaml]: Fixed

File: Hunting Queries/FileExecutionWithOneCharacterInTheName.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+id: 299472c4-8382-4c5b-82d9-718cda193393
+name: Execution of File with One Character in the Name 
+description: |
+   'This query detects execution of files with one character in the name (e.g, a.exe, 7.ps1, g.vbs etc.). 
+   Normally files that are executed have more characters in the name and this can indicate a malicious file.
+   Ref: https://www.mandiant.com/resources/tactics-techniques-procedures-associated-with-maze-ransomware-incidents'
+severity: Medium
```
Comment (aprakash13) [inline Hunting Queries/FileExecutionWithOneCharacterInTheName.yaml:7]: Please remove the Severity as this is a Hunting query

**Review by aprakash13 — CHANGES_REQUESTED** [review]

**Review by aprakash13 — APPROVED** [review]
Thanks @vpaschalidis

**Pull request merged**